src/App.js

```diff
@@ -1,9 +1,7 @@
 import React, { Component } from "react";
 import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import ImageAnnotation from "./components/ImageAnnotation";
-
-import UploadImageFile from "./components/images_uploading/UploadImageFile";
```

Comment:
I changed the file structure drastically. Now for image uploading, the structure is 'App.js' -> 'images_uploading\ImageManagement.js' -> 'images_uploading\DisplayImageList.js' & 'images_uploading\UploadImageFile.js'